telco-hub/configuration/reference-crs-kube-compare/required/registry/registry-ca.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "-40"
+  name: registry-ca
+  namespace: openshift-config
+{{- if .data }}
+data:
+{{- .data | toYaml | nindent 2 }}
```

Comment:
```suggestion
data:
  # important: keep the format "url..port"
{{- .data | toYaml | nindent 2 }}
```
If the file in the source-crs file has a comment, here that comment should be set as well... both files should be almost identical (except the templatization code) so the compare tool is happy... or you may decide to remove the comment from both files, up to you...

Reply:
but I am not receiving any error/warning from the tool. Where can I see taht?